### ISSUE TITLE
The git submodules init command throws an error

### ISSUE THREAD

**Issue by shantanu2704** [Automattic/vip-go-compatibility-scanner#11]
Source :https://github.com/Automattic/vip-go-compatibility-scanner/blob/master/README.md#usage-for-a-single-repository

After the git repo is cloned, we need to `cd` into the newly created directory to be able to run 

```
git submodule init && \
git submodule update --recursive && \
```

Otherwise we get an error:

```
fatal: not a git repository (or any of the parent directories): .git
```

**Comment by gudmdharalds** [comment]
Thank you for pointing this out. This is my mistake.

I will fix this.

**Comment by gudmdharalds** [comment]
Fixed in ea341fc.
